vars/test.yml

```diff
@@ -133,14 +133,6 @@ vm_password: "B1gd3m0z"
 # Default value is 'auto'.
 # vm_python: "auto"
 
```

Comment:
Then `vm_username` part description can be improved accordingly. For existing VM, must config "root" user, for new created VM, configured non-root user will be created, while run testing the "root" user is used, user passwords are the same.
Others look good to me. Thanks.